src/components/header/index.tsx

```diff
@@ -1,11 +1,43 @@
-import React from "react";
+import { Alignment, Button, Icon, Navbar as BPNavbar } from "@blueprintjs/core";
+import * as React from "react";
+import { Link } from "react-router-dom";
+import HeaderPatientName from "components/header/headerPatientName";
+
 import "./style.css";
 
-const Header = () => {
+interface Props {
+  patientId: string;
+}
+
+const Header = ({ patientId }: Props) => {
   return (
-    <div className="App-header">
-      <h2 className="App-title">Welcome to PatientVisualizer !</h2>
-    </div>
+    <BPNavbar id="navbar" className="bp3-dark">
+      <BPNavbar.Group align={Alignment.LEFT}>
+        <BPNavbar.Heading>
+          <Link
+            style={{ textDecoration: "none", color: "white", display: "flex" }}
+            to="/"
+          >
+            <img
+              id="logoNavbar"
+              src={process.env.PUBLIC_URL + "/arkhn_logo_only_white.svg"}
+              alt="Arkhn"
+            />
+            <h3 id="titleNavbar">TIMELINE</h3>
+          </Link>
+        </BPNavbar.Heading>
+        {/* TODO: update replace patientId with patient name dynamically */}
+        {patientId !== "" ? <HeaderPatientName patientId={patientId} /> : ""}
```

Comment:
patientId should never be an empty string. in typescrypt you can add a question mark in your type definition if it may be null. eg: 
```
interface Props {
  patientId?: string;
}
```
then you can do `{patientId && <HeaderPatientName patientId={patientId} /> ` instead of using a ternary